src/Raven.Server/Smuggler/Documents/StreamSource.cs

```diff
@@ -498,6 +517,32 @@ await ReadObjectAsync(reader =>
                             _log.Info("Wasn't able to import the RavenDB connection strings from smuggler file. Skipping.", e);
                     }
                 }
+                
+                if (reader.TryGet(nameof(databaseRecord.AiConnectionStrings), out BlittableJsonReaderObject aiConnectionStrings) && aiConnectionStrings != null)
+                {
+                    try
+                    {
+                        foreach (var connectionName in aiConnectionStrings.GetPropertyNames())
+                        {
+                            if (aiConnectionStrings.TryGet(connectionName, out BlittableJsonReaderObject connection) == false)
+                            {
+                                if (_log.IsInfoEnabled)
+                                    _log.Info($"Wasn't able to import the RavenDB connection string {connectionName} from smuggler file. Skipping.");
+
+                                continue;
+                            }
+
+                            var connectionString = JsonDeserializationCluster.AiConnectionString(connection);
+                            databaseRecord.AiConnectionStrings[connectionName] = connectionString;
+                        }
+                    }
+                    catch (Exception e)
+                    {
+                        databaseRecord.RavenConnectionStrings.Clear();
```

Comment:
should this be `atabaseRecord.AiConnectionStrings` ?

Reply:
Fixed